File: ProgressNotification/app/src/main/java/com/akexorcist/progressnotification/ui/SplashScreenActivity.java
package com.akexorcist.progressnotification.ui;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.Button;

import com.akexorcist.progressnotification.R;

public class SplashScreenActivity extends AppCompatActivity {
    private Button btnNext;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash_screen);

        bindView();
        setupView();
    }

    private void bindView() {
        btnNext = findViewById(R.id.btn_next);
    }

    private void setupView() {
        btnNext.setOnClickListener(view -> goToMainScreen());
    }

    private void goToMainScreen() {
        startActivity(new Intent(SplashScreenActivity.this, ApplyAccountActivity.class));
    }
}

File: app/src/main/java/com/akexorcist/progressnotification/ui/SplashScreenActivity.kt
package com.akexorcist.progressnotification.ui

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.akexorcist.progressnotification.R
import kotlinx.android.synthetic.main.activity_splash_screen.*

class SplashScreenActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash_screen)

        setupView()
    }

    private fun setupView() {
        btnNext?.setOnClickListener { goToMainScreen() }
    }

    private fun goToMainScreen() {
        startActivity(Intent(this, ApplyAccountActivity::class.java))
    }
}
